app/component/SummaryRow.js

```diff
@@ -289,7 +289,6 @@ const SummaryRow = (
     if (leg.route) {
       if (
         previousLeg &&
-        !previousLeg.intermediatePlace &&
```

Comment:
What is the result of this?

Reply:
The deletion of this line was only a try and stayed in the commit by mistake.